yoti-sdk-api/src/main/java/com/yoti/api/client/identity/ShareSessionQrCode.java

```diff
@@ -10,6 +14,21 @@ public class ShareSessionQrCode {
     @JsonProperty(Property.URI)
     private String uri;
 
+    @JsonProperty(Property.EXPIRY)
+    private String expiry;
+
+    @JsonProperty(Property.POLICY)
+    private String policy;
+
+    @JsonProperty(Property.EXTENSIONS)
+    private List<Extension> extensions;
+
+    @JsonProperty(Property.SESSION)
+    private ShareSession session;
+
+    @JsonProperty(Property.REDIRECT_URI)
+    private String redirectUri;
```

Comment:
It's always preferable to use strong types (in this case `URI`

Reply:
maybe change this one too in the same class if possible
```
    @JsonProperty(Property.URI)
    private String uri;
```